.github/workflows/image-build.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
       containerfile: "./Dockerfile"
       architectures: '[ "amd64", "arm64", "ppc64le", "s390x" ]'
       extra-args: "--ulimit nofile=4096:4096"
+      pre_build_cmd: |
```

Comment:
How about adding a new env variable i.e. FORCE_NPM_9 and then explicitly branching based on that?
The `sed` is more elegant but is not visible while looking at the commands  list.
Alternatively we could use some alarmist comment above the `clean-install` command ("Attention! Attention!....")

Reply:
Point taken.  I've made some more obvious changes to force `npm@^9`.